### ISSUE TITLE
CS2 Aimbot

### ISSUE THREAD

**Issue by Svxy** [danielkrupinski/Osiris#4307]
Is there anyone with an aimbot function out there? or triggerbot/better esp? how come theres so little features as opposed to csgo?

**Comment by Aliorpse** [comment]
less feature+best visual=safest cheat

**Comment by Svxy** [comment]
> less feature+best visual=safest cheat

Yeah but i dont care about bans ive made like 30 accounts lol. i was just hoping someone might have added an aimbot feature and could send me the source or something.

**Comment by Deskilling** [comment]
okö
